nss-cohort-36/bangazon-workforce-mgt-illin-illusionists
# Add New Employee

# Description

The branch name is correct. No worries.

Relates to issue # [**18**]

## Type of change

_Please delete options that are not relevant._


## Testing Instructions

### Setup 

Ensure you are in your `/bangazon-workforce-mgt-illin-illusionists/hrapp/` directory then setup with the following commands

```bash
git fetch --all
git checkout gc-training-delete
python manage.py runserver
```

### Testing Steps
1. Click on `Employee` from the navbar
2.  Click on `Add Employee to Bangazon` from list view
3.  Expect to be routed to the Add Employee form
4. Fill the form with relevant Data.
5. Click `Create`
6.  Expect to be redirected back to Employee list view
7. Expect to see the newly added employee


## Checklist:
